app/views/idv/inherited_proofing_cancellations/new.html.erb

```diff
@@ -0,0 +1,55 @@
+<% title t('titles.inherited_proofing.cancellation_prompt') %>
```

Comment:
Would it have been possible to reuse the same cancellation screen as in IdV, rather than creating a copy? I worry about keeping these in sync.